plugins/org.jboss.tools.jst.js.bower/plugin.xml

```diff
@@ -57,6 +57,52 @@
          </contextualLaunch>
       </shortcut>
    </extension>
+
+   <extension point="org.eclipse.ui.commands">
+      <command
+            name="bower update"
+            description="Bower Update"
+            categoryId="org.eclipse.debug.ui.category.run"
+            id="org.jboss.tools.jst.js.bower.command.update"
+            >
+      </command>
+      
+      <!--keyBinding
+            keySequence="M3+M2+D E"
+            contextId="org.eclipse.ui.globalScope"
+            commandId="com.example.ExampleLaunchShortcutId.debug"
+            keyConfigurationId="org.eclipse.ui.defaultAcceleratorConfiguration">
+      </keyBinding-->
+   </extension>    
+
+<extension
+ point="org.eclipse.ui.commandImages">
+ <image
+  commandId="org.jboss.tools.jst.js.bower.command.update"
+  icon="icons/bower_16.png" />
```

Comment:
I don't know why but the icon is not shown up in Quick Access. Requires investigation.